### DISCUSSION TITLE
project contribution

### DISCUSSION THREAD

**Discussion by ejangphoebe00** [LesFruitsDefendus/saskatoon-ng#209] [Outreachy]
Hello guys i am phoebe , i want to know how to move forward with the project contribution i have just joined and want to know if solving issues from the issue tracker are the project contributions we are supposed to make and how to contact mentors after making a pull request. and do we have to wait for the PR to be reviewed before moving on to the next issue. thanx

**Comment by tristanlatr** [comment]
Hello @ejangphoebe00,

Sorry the the delay. 

> want to know if solving issues from the issue tracker are the project contributions we are supposed to make

Yes this is true, also if you have other ideas of enhancements, don't hesitate to open more issues.

> how to contact mentors after making a pull request

We should receive notifications when people are opening PRs, so me or @2ynn must review your pull request, accept it, then we can more forward and merge the PR. If you opened a PR and your are not sure if you should continue working on it, just tag me in the comments so I get a new notification. 

> and do we have to wait for the PR to be reviewed before moving on to the next issue. 

If you'r confident you can work on several issues at the same time, please do so, bu tit might be overwhelming. 

Hope this answers your questions,

Talk to you later,
